jsonschema-validation.xml

```diff
@@ -550,6 +550,20 @@
                 </t>
             </section>
 
+            <section title="propertyNames">
+                <t>
+                    The value of "propertyNames" MUST be a valid JSON Schema.
+                </t>
+                <t>
+                    If the instance is an object, this keyword validates if every property name in the instance
+                    validates against the provided schema.
+                    Note the property name that the schema is testing will always be a string.
+                </t>
+                <t>
+                    A missing keyword is the same as an empty schema, or a schema testing that the type is "string".
```

Comment:
I can say "A missing keyword exhibits the same behavior as an empty schema" or something like that... The thing is it has to be informative, since it's noting something that's already true.